packages/calcite-components/src/components/text-area/text-area.tsx

```diff
@@ -161,6 +161,11 @@ export class TextArea
   /** Accessible name for the component. */
   @property() label: string;
 
+  /**
+   * When `true`, prevents input beyond the maximum length, mimicking native `<textarea>` behavior.
+   */
+  @property() limitText = false;
```

Comment:
should probably be reflected and a reflects test added

Reply:
Good catch. Done!